utils.py

```diff
@@ -78,13 +78,18 @@ def shuffle(lst):
         lt.append(lst[r]); l1.remove(lst[r])
     return lt
 
-def centre_blit(screen,img,(cx,cy),angle=0): # rotation is clockwise
+#The word 'cord' used later on in various variables refers to the word 'co-ordinates'
+
+def centre_blit(screen,img, coordinates,angle=0): # rotation is clockwise
+    (cx,cy) = coordinates
     img1=img
     if angle!=0: img1=pygame.transform.rotate(img,-angle)
     rect=img1.get_rect()
     screen.blit(img1,(cx-rect.width/2,cy-rect.height/2))
 
-def text_blit(screen,s,font,(cx,cy),(r,g,b),shadow=True):
+def text_blit(screen,s,font, coordinates, color,shadow=True):

```

Comment:
Mix of using spaces or not using spaces; please use existing style in the file, unless all such lines are changed in which case uses a space after comma.